recipes/libcufile/build.bat

```diff
@@ -0,0 +1,17 @@
+if not exist %PREFIX% mkdir %PREFIX%
```

Comment:
We don't need this Windows build script, but we do need a linux one to set up the targets directory: 
https://github.com/conda-forge/staged-recipes/pull/21723/files#diff-e5bafa1dffa25db6c210c96887404cd7168b4c02772daf9fa5cbedb52b5fbd0e
The pkgconfig name will need to be updated to "cufile*.pc"
And the ppc64le line can be dropped